### ISSUE TITLE
ValueError: Unknown layer:name

### ISSUE THREAD

**Issue by adamkc** [rstudio/tensorflow#321]
I am trying to load a model file and I receive an error message: 

```
> model <- keras::load_model_hdf5("ModelFile.h5")
Error in py_call_impl(callable, dots$args, dots$keywords) : 
  ValueError: Unknown layer:name

Detailed traceback: 
  File "C:\Users\ADAMCU~1\AppData\Local\CONTIN~1\ANACON~1\envs\KERAS-~1\lib\site-packages\tensorflow\python\keras\engine\saving.py", line 229, in load_model
    model = model_from_config(model_config, custom_objects=custom_objects)
  File "C:\Users\ADAMCU~1\AppData\Local\CONTIN~1\ANACON~1\envs\KERAS-~1\lib\site-packages\tensorflow\python\keras\engine\saving.py", line 306, in model_from_config
    return deserialize(config, custom_objects=custom_objects)
  File "C:\Users\ADAMCU~1\AppData\Local\CONTIN~1\ANACON~1\envs\KERAS-~1\lib\site-packages\tensorflow\python\keras\layers\serialization.py", line 64, in deserialize
    printable_module_name='layer')
  File "C:\Users\ADAMCU~1\AppData\Local\CONTIN~1\ANACON~1\envs\KERAS-~1\lib\site-packages\tensorflow\python\keras\utils\generic_utils.py", line 173, in deserialize_keras_object
    list(custom_objects.items())))
  File "C:\Users\ADAMCU~1\AppData\Local\CONTIN~1\ANACON~1\envs\KERAS
```

Keras is fully updated to 2.2.4 in the conda environment:

```
(keras-gpu) F:\Adam C\project>conda update keras
Collecting package metadata: done
Solving environment: done

# All requested packages already installed.
```

```
> use_condaenv("keras-gpu",required=TRUE)
> py_config()
python:         C:\Users\adamc\AppData\Local\Continuum\anaconda3\envs\keras-gpu\python.exe
libpython:      C:/Users/adamc/AppData/Local/Continuum/anaconda3/envs/keras-gpu/python36.dll
pythonhome:     C:\Users\ADAMCU~1\AppData\Local\CONTIN~1\ANACON~1\envs\KERAS-~1
version:        3.6.6 |Anaconda, Inc.| (default, Jun 28 2018, 11:27:44) [MSC v.1900 64 bit (AMD64)]
Architecture:   64bit
numpy:          C:\Users\ADAMCU~1\AppData\Local\CONTIN~1\ANACON~1\envs\KERAS-~1\lib\site-packages\numpy
numpy_version:  1.15.4
tensorflow:     C:\Users\ADAMCU~1\AppData\Local\CONTIN~1\ANACON~1\envs\KERAS-~1\lib\site-packages\tensorflow\__init__.p

NOTE: Python version was forced by use_python function
```
  It seems others have had this problem and it should have been fixed in 2.2.4: https://github.com/keras-team/keras/issues/11617#issue-379502677

Any suggestions on where to go from here?

**Comment by adamkc** [comment]
I resolved the issue by updating CUDA (10.0) and cuDNN (7.4).

**Comment by edoson** [comment]
Interested in this as well. @adamkc the CUDA solution is not relevant for those using CPU I believe..
